pages/team.js

```diff
@@ -0,0 +1,95 @@
+// eslint-disable-next-line no-restricted-imports
+import React from 'react';
+import { getTeamMembersPromise } from 'common/constants/api';
+import { s3 } from 'common/constants/urls';
+import QuoteBanner from 'components/_common_/HeroBanner/HeroBanner';
+import Section from 'components/_common_/Section/Section';
+import TeamCard from 'components/_common_/TeamCard/TeamCard';
+import styles from './styles/team.css';
+
+export default class team extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      boardMembers: null,
+      staffMembers: null,
+    };
+  }
+
+  async componentWillMount() {
```

Comment:
Data fetching should occur in cDM (`componentDidMount()`).

It _sounds_ like it should happen on cWM (`componentWillMount()`), but
- cWM is actually deprecated as of React 16.3
- An API call with fetch or axios inside componentWillMount will not return before the first render. This means the component will render with empty data at least once. Basically, cDM fetches guarantee only one rerender, but with cWM you could have 2.